src/main/scala/com/microsoft/partnercatalyst/fortis/spark/analyzer/timeseries/ComputedTileToKmeansComputedAnomaly.scala

```diff
@@ -0,0 +1,52 @@
+package com.microsoft.partnercatalyst.fortis.spark.analyzer.timeseries
+
+import java.sql.Timestamp
+
+import com.datastax.spark.connector._
+import com.microsoft.partnercatalyst.fortis.spark.dto.{ComputedAnomaly, ComputedTile, MLModel, TopicCount}
+import org.apache.spark.ml.clustering.KMeansModel
+import org.apache.spark.rdd.RDD
+import org.apache.spark.sql.SparkSession
+import org.joda.time.Instant
+
+object ComputedTileToKmeansComputedAnomaly {
+
+  def apply(tiles: RDD[ComputedTile], session: SparkSession): RDD[ComputedAnomaly] = {
+    tiles.flatMap(tile=>{
+      models(tile, session).flatMap(model => {
+        val parameters = ComputedAnomalyKmeansModelParameter.parametersForAnalysisByKey(model.key)
+        val topicCount = TopicCount(tile)
+        val input = ComputedAnomalyKmeansInputRecord(topicCount, parameters)
+        val kmeansModel = model.dataAs[KMeansModel]()
+        val tileDF = session.createDataFrame(Seq(input))
+
+        val modelCost = model.metadata("cost").toDouble
+        val computedCost = kmeansModel.computeCost(tileDF)
+        if (computedCost < computedCost) Seq()
```

Comment:
I am a little confused by this if statement's condition: `computedCost < computedCost`. I believe this would always be false.